app/init.php

```diff
@@ -1049,6 +1049,9 @@ function (mixed $value) {
         }
     }
 
+    $dbForProject->setMetadata('user', $user->getId());
```

Comment:
Why not do it directly when we create the DB instance and have `user` as a dependency?

Reply:
We inject the db's when creating the user resource so we'd end up with a cyclic dependency